src/main/java/frc/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -32,32 +65,60 @@
           // Back right
           new Translation2d(-Dims.TRACKWIDTH_METERS / 2.0, -Dims.WHEELBASE_METERS / 2.0));
 
-  private final AHRS navx = new AHRS(SPI.Port.kMXP, (byte) 200); // NavX connected over MXP
+  /**
+   * The SwerveDriveOdometry class allows us to estimate our vehicle position over time using the
+   * power of :sparkles: math :sparkles:.
```

Comment:
Do ``:sparkles:`` render in doccomments on vscode?

Reply:
Unfortunately, no, I'll remove these 😔